assignments/lambda-classes.js

```diff
@@ -1 +1,86 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(attributes){
+        this.name = attributes.name;
+        this.age=attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+    }
+    greet () {
+        return `Hello my name is ${this.name}, I am from ${this.location}`;
+    }
+} // parent
+
+class Instructor extends Person {
+    constructor(instructorAttributes){
+        super(instructorAttributes);
+        this.specialty = instructorAttributes.specialty;
+        this.favLangauge = instructorAttributes.favLangauge;
+        this.catchPhrase = instructorAttributes.catchPhrase;
+    }
+    demo (subject) {
+        return `Today we are learning about ${subject}`;
+    }
+    grade (student, subject) {
+       return  `${student.name} receives a perfect score on ${subject}`;
+    }
+} //instructor
+ class Student extends Person {
+     constructor (studentAttributes) {
+         super(studentAttributes);
+         this.previousBackground = studentAttributes.previousBackground;
+         this.className = studentAttributes.className;
+         this.favSubjects = studentAttributes.favSubjects;
+     }
+     listsSubjects () {
+         return this.favSubjects;
+     }
+     PRAssignment (subject) {
+return `${this.name} has submitted a PR for ${subject}`;
+     }
+     
+ } //students
+ class Project_Manager extends Person {
+constructor (project_managerAttributes){
+    super(project_managerAttributes);
+    this.gradClassName = project_managerAttributes.gradClassName;
+    this.favInstructor= project_managerAttributes.favInstructor;
+}
+standUp (slack_channel) {
```

Comment:
if you're passing in the variable as slack_channel you should reference it as such in the template literal ${slack_channel}